src/modules/licences/controller.js

```diff
@@ -56,18 +70,48 @@ const wrapData = data => ({
   data
 });
 
+const addEarlistEndDate = licence => {
```

Comment:
addEarliestEndDate

Reply:
Bah, I did that so many other times as well.